xml/bp_troubleshooting_tips.xml

```diff
@@ -136,19 +134,16 @@ osd_scrub_load_threshold = 0.25
    After you complete the maintenance, start OSDs again:
   </para>
 
-<screen>
-  &prompt.sminion;systemctl start ceph-<replaceable>FSID</replaceable>@osd.<replaceable>OSD_NUMBER</replaceable>.service
-</screen>
+<screen>&prompt.sminion;systemctl start ceph-<replaceable>FSID</replaceable>@osd.<replaceable>SERVICE_ID</replaceable>.service</screen>
 
   <para>
    After OSD services are started, unset the cluster from
    <literal>noout</literal>:
   </para>
 
-<screen>
-&prompt.cephuser;ceph osd unset noout
-</screen>
+<screen>&prompt.cephuser;ceph osd unset noout</screen>
  </sect1>
+<!-- 2020-09-22 tbazant: i suggest removing the following whole section -->
```

Comment:
@smithfarm what do you think of this suggestion? The following section online is: https://susedoc.github.io/doc-ses/master/html/ses-troubleshooting/storage-tips.html#Cluster-Time-Setting 

Sounds like a reasonable idea to me.

Reply:
:+1: